composer.json

```diff
@@ -29,7 +29,7 @@
         "doctrine/doctrine-bundle": "^2.3",
         "doctrine/orm": "^2.5",
         "doctrine/persistence": "^2.0",
-        "sonata-project/admin-bundle": "^4.0@alpha",
+        "sonata-project/admin-bundle": "^4.0@dev",
```

Comment:
you will also need to conflict with previous alphas

Reply:
"sonata-project/admin-bundle": "<=4.0.0-alpha-1" ?